### DISCUSSION TITLE
Add option to select multiple servers for restriction on pack sales and not just one server or whole cluster

### DISCUSSION THREAD

**Discussion by Ian-Highlander** [gameserverapp/Platform#811] [Suggestions]
I'd like the option to be able to select more than one server to restrict packs to without having to duplicate the packs as I do currently or use the block other servers options.

We have the option to select whole cluster or individual servers currently but not more than one server individually (unless I'm missing it). 

Specific situation for me is that I have a test server that will run duplicates of maps on my live servers and I want to easily be able to target the same things to both the live and test server for that map.

I know you can choose a whole cluster and then block other servers but would have thought it simpler to just have a tick which servers you want to allow option as well. This is going to get harder to keep maintained going forward as new maps come on stream for ASA.

![image](https://github.com/gameserverapp/Platform/assets/109648228/d9e2ccc2-7e3a-4b30-99c3-014c94b97437)

**Comment by IIXx6IIxX** [comment]
Great suggestions, would be great to have this!

**Comment by mvaessen** [comment]
duplicate of #577

**Comment by Ian-Highlander** [comment]
Yes it is but the choice you've given in answer to that one doesn't resolve the particular issue this refers to.

**Reply by mvaessen** [reply]
I might have misunderstood your suggestion, it seemed very similar to me. Feel free to add missing info on the other post though.

Just trying to keep the discussion board somewhat structured and clean from duplicate / similar suggestions, so it remains workable 😀

**Reply by Ian-Highlander** [reply]
> I might have misunderstood your suggestion, it seemed very similar to me. Feel free to add missing info on the other post though.
> 
> Just trying to keep the discussion board somewhat structured and clean from duplicate / similar suggestions, so it remains workable 😀

![image](https://github.com/user-attachments/assets/22dd1f37-27d3-42da-8737-81791dabfc6d)
